tests/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -12,11 +18,58 @@
     FORWARD_MODEL_STATE_RUNNING,
 )
 
-from .ensemble_evaluator_utils import send_dispatch_event
+from .ensemble_evaluator_utils import TestEnsemble, send_dispatch_event
 
 
-def test_new_monitor_can_pick_up_where_we_left_off(evaluator):
-    evaluator._start_running()
+@pytest.fixture(name="evaluator_handle")
+def start_ee_in_thread(make_ee_config):
+    msg_queue = queue.Queue()
+    ee_async = None
+
+    def _ee_start():
+        async def start_ee_async():
+            nonlocal ee_async, msg_queue
+            ensemble = TestEnsemble(0, 2, 2, id_="0")
+            ee_async = EnsembleEvaluatorAsync(ensemble, make_ee_config(), 0)
+            msg_queue.put(ee_async)
+            run_task = asyncio.create_task(
+                ee_async.run_and_get_successful_realizations()
+            )
+            await ee_async._server_started.wait()
+            msg_queue.put("STARTED")
+            num_realizations = await run_task
+            msg_queue.put(num_realizations)
+
+        loop = new_event_loop()
+        asyncio.set_event_loop(loop)
```

Comment:
I think we might be able to remove `asyncio.set_event_loop(loop)` completely, as we are directly referencing the new event loop below anyways.